Nakama/IClient.cs

```diff
@@ -54,6 +54,15 @@ public interface IClient
         /// </summary>
         string ServerKey { get; }
 
+        /// <summary>
+        /// Received a new session after the current one had expired.
```

Comment:
@novabyte I think this event will still fire even if `SessionRefreshAsync` is manually invoked by the user, regardless of whether `autoRefreshSession` is enabled or the current session had expired. I think that's fine but then this comment and remark would need to be adjusted.